### ISSUE TITLE
Support rounding of columns with numeric values

### ISSUE THREAD

**Issue by heralden** [intermine/im-tables-3#100]
If a column is numeric, a button should appear in its column header that allows rounding of the numbers to a chosen decimal point. This rounding will only apply to the values of this column, on this specific table.

**Comment by AdrianBZG** [comment]
Also, an option to round all numbers in all columns by default would be helpful, otherwise the user would need to be clicking the round button in every column each time a page is accessed.

**Comment by heralden** [comment]
The team thought that having it by default wouldn't be a good idea.

I guess the middle ground would be to have it global then. I think there are some different approaches:
- Should it be configured by a user to apply to all instances of that column in all im-tables in Bluegenes?
- Should a mine admin configure a default, i.e. they set which columns (by path) should be rounded and to which decimal?

I'm guessing the second one is what you want?

**Comment by AdrianBZG** [comment]
Yes, the second one sounds like a good approach, although that would mean specifying columns path by path. In some sources I have more than 20 columns with decimal numbers, so that's why I was suggesting to also have some kind of global rounding option. But it's fine to go with the second option you mention.

**Comment by heralden** [comment]
Do those 20 columns point to the same class? If so we could instead specify rounding for classes.

Edit: The rounding should be specified for an **attribute** of a class.

**Comment by AdrianBZG** [comment]
Yes, they belong all to the same class

**Comment by heralden** [comment]
Sounds good! We'll have to figure out where this should be defined, but I'm guessing it will be in a property file on the Intermine backend.

**Comment by danielabutano** [comment]
We can use the[ webconfig_model.xml](https://intermine.readthedocs.io/en/latest/webapp/properties/webconfig-model/) file, in the mine instance, to add the new property _rounding_
